contents/handbook/people/compensation.mdx

```diff
@@ -69,11 +72,23 @@ As we do these much more frequently than regular companies, team members should
 
 ### Pay review process
 
+Pay at PostHog is reviewed much more often at most other companies - at least 2 to 3 times a year.
+
+To make sure everyone has an equal chance at getting a payrise we do not factor in how much someone complains about their pay. When increasing pay we only look at our calculator and performance.
+
+If someone is a high performer they will be paid well.
+
+<!-- Our goal is to increase their pay as frequently as we can. This an example of being inclusive - rather than rewarding the loudest voices. -->
+
 We use an internal compensation calculator spreadsheet to track salaries, underlying benchmarks and increase dates. We use our [Compensation Calculator](/handbook/people/compensation) as the basis of any compensation reviews. 
 
 Any increases will be communicated by James and Tim, as they are the only people responsible for communicating pay rises, as compensation is not a [manager's responsibility](/handbook/company/management) at PostHog. Eltje will follow up with an email confirmation and make the pay increases in time for the next payroll. 
 
-You will only hear from them if there has been a change in your pay, not if it is staying the same. 
+You will only hear from them if there has been a change in your pay, not if it is staying the same.
+
+We do not review everyone's pay on the same day, and we do not share when we decided not to increase your pay - deliberately. This is to reduce pressure around the process.
+
+Very frequently we are unable to increase pay - we don't want people to feel disappointed, sometimes it just takes a little while to progress to the next stage of your career. 
```

Comment:
I think this wording could potentially be alienating for candidates who are newer in their career and want to work with us. Do we really need this line at all? I'm not sure it's adding much other than calling anyone who doesn't see a compensation increase for whatever reason. Thanks for updating this to clarify!

Reply:
Agree - to me the point is that more frequent reviews -> more frequent no change. You shouldn't expect a quarterly increase in your pay as it's not realistic. Rather than take the line out entirely, I've amended it as I think it's helpful to set expectations.